test/core/select.wast

```diff
@@ -251,7 +315,7 @@
 (assert_return (invoke "as-br_table-last" (i32.const 1)) (i32.const 2))
 
 (assert_return (invoke "as-call_indirect-first" (i32.const 0)) (i32.const 3))
-(assert_return (invoke "as-call_indirect-first" (i32.const 1)) (i32.const 2))
+;;(assert_return (invoke "as-call_indirect-first" (i32.const 1)) (i32.const 2))
```

Comment:
Was it intentional that this was commented out?

Reply:
Hm, probably not, but I can't trace back where it came from. Mind creating a PR to reactivate it?

Reply:
No problem, I've done so here: https://github.com/WebAssembly/spec/pull/1323